hdfs3/core.py

```diff
@@ -383,11 +383,26 @@ def exists(self, path):
         return out == 0
 
     def chmod(self, path, mode):
-        """ Mode in numerical format (give as octal, if convenient) """
-        imode = mode_numbers[mode]
+        """Change access control of given path
+
+        Exactly what permissions the file will get depends on HDFS
+        configurations.
+
+        Parameters
+        ----------
+        path : string
+            file/directory to change
+
+        mode : integer
+            As with the POSIX standard, each octal digit refers to
+        user-group-all, in that order, with read-write-execute as the
+        bits of each group. E.g.,
+        - 0o777 -> permissive;
+        - 0o711 -> all permission for owner, read-only for others;
+        - 0o100 -> read-only for owner, nothing for others."""
```

Comment:
To work well with sphinx documentation the entire description of the parameter `mode` needs to be indented four spaces.  Can we briefly define or point to the posix standard in the parameters section and then include an Examples section just below:

```
Examples
-------
>>> hdfs.chmod('/path/to/file', 0x777)  # make read/writeable to all # doctest: +SKIP
>>> hdfs.chmod('/path/to/file', 0x700)  # make read/writeable only to user # doctest: +SKIP
```

Can we support strings as well?

```python
In [1]: s = '777'
In [2]: 8**2 * int(s[0]) + 8**1 * int(s[1]) + 8**0 + int(s[0])
Out[2]: 512
```